Descent3/loadstate.cpp

```diff
@@ -1603,423 +1528,4 @@ int LGSSnapshot(CFILE *fp) {
   }
 
   return bm_handle;
-}
-
-//@@	// movement info.
-//@@		switch (op->movement_type)
-//@@		{
-//@@		case MT_SHOCKWAVE:
```

Comment:
do you have any idea what this whole code block could have been for?

Reply:
If you search `@` you'll find a lot of files with commented blocks like this one. In this block, it looks like old loading code that was replaced.